openedx/core/djangoapps/kafka_consumer/apps.py

```diff
@@ -0,0 +1,6 @@
+"""App for consuming Kafka events. Comprises a management command for listening to a topic and supporting methods."""
```

Comment:
Please add a `/kafka_consumer/README.txt`. Some things to explain:
1. Purpose of app.
2. Temporary nature of app. Note that some or all of this should be pluggable, and that Kafka should not be a future requirement. If it makes sense to write a small ADR under `/kafka_consumer/docs` with more details, feel free. Note: the rest of the ADR decisions you wanted to write could also live here for now.